scrapi/base/__init__.py

```diff
@@ -241,14 +242,8 @@ def normalize(self, raw_doc):
                 'ns0:header/ns0:setSpec/node()',
                 namespaces=self.NAMESPACES
             )
-            approved = False
-            for item in set_spec:
-                item_mod = item.replace('publication:', '')
-                if item_mod in self.approved_sets:
-                    approved = True
-                else:
-                    logger.info('Series {} not in approved list'.format(item))
-            if not approved:
+            if not set([x.replace('publication:', '') for x in set_spec]).intersection(self.approved_sets):
```

Comment:
It may be better to break this into a function and expand it to more lines, since it seems to be pretty difficult to reason about even at the moment